assets/customer.css

```diff
@@ -386,8 +386,7 @@
 }
 
 .activate button[name='decline'],
-.addresses li > button,
-.addresses form button[type] {
+.address-form-fields button[type] {
```

Comment:
I'm not sure whether this is an old bug or something newly introduced, but once you submit the form with an invalid country, all the form styling disappears. The form inputs and buttons don't seem to have any styles applied, and this makes it very difficult to fill out the form with the correct information and submit it.
<img width="604" alt="styles missing" src="https://user-images.githubusercontent.com/22390758/201385603-711c73e0-f923-46bd-8670-77f65639cdc8.png">

Reply:
Ok

Reply:
> I'm not sure whether this is an old bug or something newly introduced, but once you submit the form with an invalid country, all the form styling disappears. The form inputs don't have borders and the buttons don't have a background colour so they look like plain text, which makes it very difficult to fill out the form with the correct information and submit it. If this is an old bug, feel free to ignore it, and we can create a new issue.
> 
> 
> 
> <img width="604" alt="styles missing" src="https://user-images.githubusercontent.com/22390758/201385603-711c73e0-f923-46bd-8670-77f65639cdc8.png">
> 
>